paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -169,7 +176,8 @@ internal fun Wallet(
                     modifier = Modifier
                         .fillMaxWidth()
                         .padding(top = 6.dp)
-                        .requiredHeight(48.dp),
```

Comment:
What’s this about? The Google Pay button is less high than the Link button. I don’t believe that we want that.

Reply:
The Google Pay button has an internal padding, which we cannot remove.

Reply:
That shouldn’t influence how large the button looks. Let’s keep the existing Link button height of 48dp and set the Google Pay button height to account for its internal padding (48dp + 2 * 4dp).